azkaban-web-server/src/web/js/azkaban/view/flow-execute-dialog.js

```diff
@@ -496,18 +498,14 @@ var disableFinishedJobs = function(data) {
       node.status = "READY";
       node.disabled = true;
     }
-    else if (node.status == "SUCCEEDED" || node.status=="RUNNING") {
+    else if (node.status == "SUCCEEDED" || node.missing) {
```

Comment:
I don't quite understand this statement. Now that node is SUCCEEDED, why we have to disable it?